File: js/test/print_date.test.js
import PrintDate from '../src/print_date';
import Calendar from '../src/calendar';
import Printer from '../src/printer';

describe('PrintDate', function () {
  it('change_this_name', function () {
    let printDate = new PrintDate(new Calendar(), new Printer());

    printDate.printCurrentDate();

    // I don't know how to test it
  });
});

File: csharp/test/PrintDateTest.cs
using NSubstitute;
using PrintDate;

namespace KataPrintDate.Tests
{
    public class PrintDateTest
    {
        [Fact]
        public void think_a_good_name_and_change_it()
        {
            var printDate = new PrintDate.PrintDate(new Calendar(), new Printer());

            printDate.PrintCurrentDate();

            // I don't know how to test it 
        }
    }
}
